elliott/elliottlib/util.py

```
@@ -734,3 +736,254 @@ def get_advisory_boilerplate(runtime, et_data, art_advisory_key, errata_type):
+                            img_url,
+                        )
+                        related_images.append(transformed_url)
+                    else:
+                        related_images.append(img_url)
+
+                    related_images = list(set(related_images))
+                    logger.warning(f"Extracted {len(related_images)} unique images from catalog.json")
+                else:
+                    raise RuntimeError(
+                        f"Neither related-images.json nor catalog.json found in pulled artifact. Available files: {pulled_files}"
+                    )
```

Ruff (0.14.4)

Abstract `raise` to an inner function

(TRY301)